packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ElementType } from 'react';
+import classNames from 'classnames';
+import { useBreadcrumbsStyleProps } from './useBreadcrumbsStyleProps';
+import { SpiritBreadcrumbsProps } from '../../types';
+import { useStyleProps } from '../../hooks/styleProps';
+
+const defaultProps = {
+  items: [],
+};
+
+export const Breadcrumbs = <T extends ElementType = 'nav'>(props: SpiritBreadcrumbsProps<T>): JSX.Element => {
```

Comment:
In https://github.com/lmc-eu/spirit-design-system/pull/431/files#diff-208644f2adb988532a0e8f1ac1350099cff104a6d3cfce2a8a7f407cb2fa5641 you are also defining `BreadcrumbItem` and `BreadcrumbLink`. Why you do not do it here in the same way?

Reply:
I wrote the React component first but you started reading first the Twig component :-) Twig version's structure is similar to the Tabs component, that's what I was getting inspiration from. The React component is much simpler than the Twig one so I didn't want to tear it apart and just add extra redundant code just to keep the same structure. I hope I understood your comment well.

Reply:
I understand your intention but I think that you are missing a few points.

- First of all, you are reusing code and defining anchor and list two times.
- Secondly, switching the class names should be based on some prop, because it is a state of a component, in this case `<BreadcrumbItem isMobileBack={true} />` (or something similar).
- Thirdly, I think that mobile Back breadcrumb should target not root item but last - 1. User want to go back one page, not to the root.

I think that we want to have a structure like this and let the developer using this component decide how to generate breadcrumbs in an app's code.

```jsx
<Breadcrumbs>
  <BreadcrumbItem url="#" isMobile={true}>Back</BreadcrumbItem>
  <BreadcrumbItem url="#">Root</BreadcrumbItem>
  <BreadcrumbItem url="#">Category</BreadcrumbItem>
</Breadcrumbs>
```

or let the developer do

```jsx
<Breadcrumbs>
  {items.map(item => <BreadcrumbItem url={item.url}>{item.title}</BreadcrumbItem> )}
</Breadcrumbs>
```

Please, read also comment from @adamkudrna - https://github.com/lmc-eu/spirit-design-system/issues/365

Reply:
There is another use case that cannot be applied on this solution and it's called "microdata". If a developer will need to make code richer by microdata, he just can't .

After a discussion we agreed that re-done of this young component is necessary.

Component should (as its React brother component) be able to accept array of items that it renders out of the box, just by itself.

Developers have to be able to render completely different content of this component like microdata usage mentioned above.

The component was reworked and is ready for its "another first review". 

Please check it out @literat

Reply:
I think that supporting microdata now is out of scope.

Reply:
As we agreed the component lets developer to choose whether to pass breadcrumb items array or to define custom ordered list's content.

Thank you, good point that the mobile devices back link href value was wrong. It's been fixed.